jvm/src/main/java/com/microsoft/bond/protocol/VarUIntHelper.java

```diff
@@ -0,0 +1,403 @@
+package com.microsoft.bond.protocol;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * Contains helper methods for working with variable-length unsigned integer representation.
```

Comment:
Renamed "Application" to "Code" :-)

---
In reply to: [116111139](https://github.com/tstein/bond/pull/4#discussion_r116111139) [](ancestors = 116111139)